.github/compatibility-test-matrices/release-v4.3.x/incentivized-transfer-chain-a.json

```diff
@@ -1,6 +1,6 @@
 {
-  "chain-a": ["release-v5.0.x"],
-  "chain-b": ["release-v5.0.x", "v6.0.0", "v5.1.0", "v4.2.0", "v4.1.1"],
+  "chain-a": ["release-v4.3.x"],
+  "chain-b": ["release-v4.3.x", "v6.0.0", "v5.1.0", "v5.0.1", "4.2.0", "v4.1.1"],
```

Comment:
Do we still need to include retracted release tags?

Reply:
I wanted to leave them here to run the compatibility tests and then remove them once we tag the new releases.